### PR TITLE
add sim colours

### PR DESCRIPTION
**Pull request by srisurya95**

### CODE REVIEW

File: res/values/colors.xml [hunk 1]
```diff
@@ -104,6 +104,5 @@
         <item>#B93221</item>
     </array>
 
-    <!-- Ripple color used over light backgrounds. -->
```
Comment (mkodekar) [inline res/values/colors.xml]: Get this back

File: src/com/android/incallui/InCallActivity.java [hunk 2]
```diff
@@ -575,6 +575,7 @@ private void internalResolveIntent(Intent intent) {
                     }
                 }
 
+<<<<<<< HEAD
```
Comment (mkodekar) [inline src/com/android/incallui/InCallActivity.java]: Remove this

File: res/values/colors.xml (hunk 1 above)
Comment (srisurya95) [inline res/values/colors.xml]: I didnt remove it @mkodekar check the code

File: src/com/android/incallui/InCallActivity.java [hunk 3]
```diff
@@ -586,6 +587,24 @@ private void internalResolveIntent(Intent intent) {
                 // is no way of making it (i.e. no valid call capable accounts)
                 if (InCallPresenter.isCallWithNoValidAccounts(call)) {
                     TelecomAdapter.getInstance().disconnectCall(call.getId());
+=======
```
Comment (mkodekar) [inline src/com/android/incallui/InCallActivity.java]: Fix this too

File: src/com/android/incallui/InCallActivity.java (hunk 2 above)
Comment (srisurya95) [inline src/com/android/incallui/InCallActivity.java]: going to fix

File: src/com/android/incallui/InCallActivity.java [hunk 4]
```diff
@@ -586,6 +587,24 @@ private void internalResolveIntent(Intent intent) {
...
+                 * to set the PhoneAccount during the PENDING_OUTGOING state.
+                 */
+                if (call != null && !isEmergencyCall(call)) {
+                    final List<PhoneAccountHandle> phoneAccountHandles = extras
+                            .getParcelableArrayList(android.telecom.Call.AVAILABLE_PHONE_ACCOUNTS);
+                    if (call.getAccountHandle() == null &&
+                            (phoneAccountHandles == null || phoneAccountHandles.isEmpty())) {
+                        TelecomAdapter.getInstance().disconnectCall(call.getId());
+                    }
+>>>>>>> f660ae9... In Multi-SIM cases use the color of the SIM icon for InCallUI.
```
Comment (mkodekar) [inline src/com/android/incallui/InCallActivity.java]: this one

**Comment by mkodekar** [comment]
Conventions say dont remove a comment

**Comment by mkodekar** [comment]
lets drop it buddy lets work on something else well i am sure something is very much fucked up in the inCallui my phone reseted thrice after hour long calls my mistake i didnt concentrate more on this stuffs but see if you are rebasing keep in mind that we dont have every thing which is present in cm and likely i would drop lollipop from end next to next week when Caf sends msm8974 merge and cm sends hals so do  nit pickings now small and minute stuffs like nav rings and button customization i am gving you more access and restruction the team

**Pull request closed without merging**